Algorithms/Other/FloodFill.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Numerics;
+
+namespace Algorithms.Other
+{
+    /// <summary>
+    /// Flood fill, also called seed fill, is an algorithm that determines and
+    /// alters the area connected to a given node in a multi-dimensional array with
+    /// some matching attribute. It is used in the "bucket" fill tool of paint
+    /// programs to fill connected, similarly-colored areas with a different color.
+    /// (description adapted from https://en.wikipedia.org/wiki/Flood_fill)
+    /// (see also: https://www.techiedelight.com/flood-fill-algorithm/).
+    /// </summary>
+    public static class FloodFill
+    {
+        private static List<ValueTuple<int, int>> neighbors = new List<ValueTuple<int, int>> { (-1, -1), (-1, 0), (-1, 1), (0, -1), (0, 1), (1, -1), (1, 0), (1, 1) };

```

Comment:
You can use List<(int xOffset, int yOffset)>

Reply:
Not sure what happened with the documentation files, maybe some quirk with my editor. I'm still getting used to value tuples, they seem to be quite convenient.